app/components/avo/actions_component.rb

```diff
@@ -2,40 +2,43 @@
 
 class Avo::ActionsComponent < Avo::BaseComponent
   include Avo::ApplicationHelper
-  attr_reader :label, :size, :as_row_control
-
-  def initialize(actions: [], resource: nil, view: nil, exclude: [], include: [], style: :outline, color: :primary, label: nil, size: :md, as_row_control: false, icon: nil)
-    @actions = actions || []
-    @resource = resource
-    @view = view
-    @exclude = Array(exclude)
-    @include = include
-    @color = color
-    @style = style
-    @label = label || I18n.t("avo.actions")
-    @size = size
-    @icon = icon
-    @as_row_control = as_row_control
+
+  prop :actions, _Array(Avo::BaseAction), default: -> { [] }
```

Comment:
Is this really an optional field? Seems like maybe it should be required, but it was optional before so I maintained that.

Reply:
I think it's best to keep it optional. We might have weird filters applied to the list and we don't want that to crash the app.